tests/bluetooth/shell/audio.conf

```diff
@@ -54,3 +54,28 @@ CONFIG_BT_MICS=y
 CONFIG_BT_MICS_AICS_INSTANCE_COUNT=1
 CONFIG_BT_MICS_CLIENT=y
 CONFIG_BT_MICS_CLIENT_MAX_AICS_INST=1
+
+CONFIG_BT_MCS=y
+CONFIG_BT_MCS_MEDIA_PLAYER_NAME_MAX=30
+CONFIG_BT_MCS_MEDIA_PLAYER_NAME="My media player"
+CONFIG_BT_MCS_ICON_URL_MAX=27
+CONFIG_BT_MCS_ICON_URL="http://www.somecompany.com"
+CONFIG_BT_MCS_TRACK_TITLE_MAX=40
+CONFIG_BT_MCS_GROUP_TITLE_MAX=40
+CONFIG_BT_MCS_SEGMENT_NAME_MAX=35
+CONFIG_BT_MCS_ICON_BITMAP_SIZE=321
+CONFIG_BT_MCC=y
+CONFIG_BT_MCC_OTS=y
+CONFIG_BT_MCC_TOTAL_OBJ_CONTENT_MEM=4096
+# TODO Check which value is sensible in the line below
+# Must be larger than any of the object sizes - icon, track, group, segments, ...
+CONFIG_BT_MCS_MAX_OBJ_SIZE=600
+CONFIG_BT_MCS_TRACK_MAX_SIZE=50
+CONFIG_BT_MCS_TRACK_SEG_MAX_SIZE=500
+CONFIG_BT_MCS_GROUP_MAX_SIZE=552
+
+# Object Transfer
+CONFIG_BT_OTS=y
+CONFIG_BT_OTS_SECONDARY_SVC=y
+CONFIG_BT_OTS_MAX_OBJ_CNT=50
+CONFIG_BT_OTC=y
```

Comment:
Hmm, we enable MCS in the shell, but we don't add the MCS module? That seems odd. I guess it is just for building purposes?

Reply:
You commented that the BabbleSim tests should have been added, and I think you are right.
When doing that, I will also add the shells.